### DISCUSSION TITLE
If I create a list when I am in a repo , the page will freeze

### DISCUSSION THREAD

**Discussion by StudyingLover** [community/community#70204] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When I meet a repo , and I am in it's page,then I create a list
![image](https://github.com/community/community/assets/68844494/3d27df7d-53bf-46e9-8bc8-685c80babefe)
After I create the list , the page will not have any change.Like this
![image](https://github.com/community/community/assets/68844494/f1f114aa-4566-467a-aae6-c7456c0f20f5)

**Comment by surya-mu** [comment, marked as answer]
> GitHub Lists is still in beta. This means that the feature is not yet fully developed and may have some bugs or limitations. 
 It is also possible that the feature could be changed or removed in the future.

Not an expert here but here you go,

I tried this in a few browsers and noticed these:

- Check if you have adblocker( This might cause some malfunctions to the repo lists)
- Check if you have starred the repo you are adding to the list ( You can currently add only starred repos, although if they're private repos or forks its not necessary it should be starred)

If you are unable to do so in above two methods I have two alternatives:

- Use Github Desktop and try adding it to the List now, make sure you have latest version of Github Desktop.
-  You can use CLI if you are familiar with it, go to your repository in Git. Then use `gh list add <list-name> <repository-name>` to try adding it to your list.


Hope this helps!

**Reply by StudyingLover** [reply]
You reminded me, I stard this repo first, and then tried to create a list.

**Reply by surya-mu** [reply]
Yep most likely this is because of its beta phase, they'll patch it hopefully
